ESMF/GOCART2G_GridComp/CA2G_GridComp/CA2G_StateSpecs.rc

```diff
@@ -97,10 +97,10 @@ category: EXPORT
  *PSOA        | kg m-2 s-1   | xy    | N     |                                | * Aerosol SOA Production
  *SMASS       | kg m-3       | xy    | N     |                                | * Aerosol Surface Mass Concentration
  *CMASS       | kg m-2       | xy    | N     |                                | * Aerosol Column Mass Density
- *EXTTAU      | 1            | xy    | N     | size(self%wavelengths_vertint) | * Aerosol Extinction AOT
- *STEXTTAU    | 1            | xy    | N     | size(self%wavelengths_vertint) | * Aerosol Extinction AOT Stratosphere
- *SCATAU      | 1            | xy    | N     | size(self%wavelengths_vertint) | * Aerosol Scattering AOT
- *STSCATAU    | 1            | xy    | N     | size(self%wavelengths_vertint) | * Aerosol Scattering AOT Stratosphere
+ *EXTTAU      | 1            | xy    | N     | size(self%wavelengths_vertint) | * Aerosol Optical Depth
+ *STEXTTAU    | 1            | xy    | N     | size(self%wavelengths_vertint) | * Stratospheric Aerosol Optical Thickness
```

Comment:
I think being consistent with either 'thickness' or 'depth' in aerosol optical depth/thickness would be better, and for other files as well.